src/Workspaces/Core/Portable/Shared/Extensions/ICompilationExtensions.cs

```diff
@@ -186,5 +186,20 @@ public static ImmutableArray<IAssemblySymbol> GetReferencedAssemblySymbols(this
 
         public static INamedTypeSymbol? SystemIDisposableType(this Compilation compilation)
             => compilation.GetTypeByMetadataName(typeof(IDisposable).FullName);
+
+        public static INamedTypeSymbol? NotNullAttribute(this Compilation compilation)
```

Comment:
These don't look used, are these just here to proactively try and make the IDE team's life better?

Reply:
They're part of the IDE change that Andrew originally committed that I left unchanged.